### ISSUE TITLE
Get error when I try install version 7.62 at Venus os v3.00~32 

### ISSUE THREAD

**Issue by testpaul999** [kwindrem/GuiMods#155]
Get error when I try install version 7.62 at Venus os v3.00~32
![IMG_4009](https://github.com/kwindrem/GuiMods/assets/35597907/8a408f16-c3ba-47d6-ad61-13243db84630)

**Comment by kwindrem** [comment]
Yes, that's true. When a Venus OS beta cycle is complete, I remove all the file sets for beta versions. You should upgrade to v3.00 (released) firmware then GuiMods will install.

**Comment by testpaul999** [comment]
😕 not a good desision, because the .32 is the last who all energymeters will use fast control loop.
The 3.00 will use this only for the EM540.
So many users will use the .32 for a wile…

regards

**Comment by kwindrem** [comment]
It is possible to go back to GuiMods v7.50 by setting the branch / tag to v7.50. But you'll loose recent enhancements.
